### PR TITLE
Amount of month in year can now be set. `monthInRow` option.

### PR DESCRIPTION
**Pull request by conmute**
I want to have 3 months in a row for month picker instead of 4.

### CODE REVIEW

**Comment by conmute** [comment]
@vlio20 I hope that simple tests will be more than enough if you need more. Just point which and why. Thank you!

File: src/app/month-calendar/month-calendar.service.spec.ts [hunk 1]
```diff
@@ -15,10 +15,11 @@ describe('Service: MonthCalendarService', () => {
   it('should check the generateYear method', inject([MonthCalendarService], (service: MonthCalendarService) => {
     const year = moment('14-01-1987', 'DD-MM-YYYY');
     const selected = moment('14-01-1987', 'DD-MM-YYYY');
-    const genYear = service.generateYear({}, year, [selected]);
+    const genYear = service.generateYear({ monthInRow: 3 }, year, [selected]);
```
Comment (vlio20) [inline src/app/month-calendar/month-calendar.service.spec.ts]: I don't mind to add it an existing test, but we should also make sure that by default we have the default amount of rows. Another thing, you have added it to a unit test where we also should add an e2e test.

File: src/app/month-calendar/month-calendar.service.ts [hunk 2]
```diff
@@ -38,9 +38,9 @@ export class MonthCalendarService {
 
   generateYear(config: IMonthCalendarConfig, year: Moment, selected: Moment[] = null): IMonth[][] {
     const index = year.clone().startOf('year');
-
-    return this.utilsService.createArray(3).map(() => {
-      return this.utilsService.createArray(4).map(() => {
+    const inRow = config.monthInRow && 12 % config.monthInRow === 0 ? config.monthInRow : 4;
```
Comment (vlio20) [inline src/app/month-calendar/month-calendar.service.ts]: please extract it to a private function.

Comment (conmute) [reply src/app/month-calendar/month-calendar.service.ts]: Ok

**Comment by conmute** [comment]
> Provide a demo (demo page) with this configuration option

Should I add a separate page in the sidebar, OR, add an input that will have values for month amount in a row?

> Add to documentation.

Ok

> Please open an issue for this PR.

Ok

**Comment by vlio20** [comment]
Add another input. No need for creating new menu item

**Comment by vlio20** [comment]
any progress?

**Pull request closed without merging**